x-pack/plugins/siem/public/components/timeline/body/index.tsx

```diff
@@ -78,7 +78,7 @@ const VerticalScrollContainer = styled.div<{
   min-width: ${({ minWidth }) => `${minWidth}px`};
 `;
 
-export const defaultWidth = 1090;
+export const DEFAULT_TIMELINE_WIDTH = 1100;
```

Comment:
Did you want `1100 //px` at the end of this like the others to indicate this is in pixels?